apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts

```diff
@@ -49,6 +49,23 @@ export const gettingstarted = {
     { name: 'Features', url: '/guides/getting-started/features' },
     { name: 'Architecture', url: '/guides/getting-started/architecture' },
     { name: 'Local Development', url: '/guides/cli/local-development' },
+    {
+      name: 'Use Cases',
+      items: [
+        {
+          name: 'OpenAI Vector Search',
+          url: '/guides/getting-started/quickstarts/openai-vector-search',
+        },
+        {
+          name: 'Subscription Payments (SaaS)',
+          url: 'https://github.com/vercel/nextjs-subscription-payments#nextjs-subscription-payments-starter',
```

Comment:
@kiwicopple we okay to link to external resources from the menu? Or should we create internal pages for these and maybe sync the readme into the docs, wdyt?

Reply:
No, I don't think we should link externally from the side menu- it's very jarring and we ideally want our users to stay on the site, especially in the "quickstart" section